save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationView.kt

```diff
@@ -474,22 +477,19 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
             it.decodeFromJsonString()
         }
 
-    private fun postImageUpload(element: HTMLInputElement) =
+    private fun postImageUpload(file: File) =
```

Comment:
These two versions of `postImageUpload()` (for users and organizations) seem almost identical.

Can we refactor them into a single function which would accept `AvatarType`?

Reply:
Fixed